.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+*.md
```

Comment:
- [ ] There is a .gitignore file listing test.md file and node_modules directory so It won't push all the independency in Github. So kindly add test.md in .gitignore file